src/main/scala/com/yotpo/metorikku/test/Tester.scala

```diff
@@ -46,7 +46,7 @@ case class Tester(config: TesterConfig) {
     val variables = params.variables
     val inputs = getMockFilesFromDir(config.test.mocks, config.basePath)
     Configuration(Option(metrics), inputs, variables, None, None, None, None, None,
-      Option(config.preview > 0), None, None,  None, Option(config.preview), None, None, None, None)
+      Option(config.preview > 0), None, None,  None, Option(config.preview), None, None, None, None, None)
```

Comment:
I think it should be `Option(true)`

Reply:
Changing to Option(false) (checks should run if you don't pass anything)